File: sub-web-model/src/main/java/org/github/base/service/impl/SysUserServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysUserEntity;
import org.github.base.mapper.ISysUserMapper;
import org.github.base.service.ISysUserService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 系统用户 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-08
 */
@Service
public class SysUserServiceImpl extends ServiceImpl<ISysUserMapper, SysUserEntity> implements ISysUserService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/SysUserServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.SysUserEntity;
import org.github.mysql.mydb.base.mapper.ISysUserMapper;
import org.github.mysql.mydb.base.service.ISysUserService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 用户表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysUserServiceImpl : ServiceImpl<ISysUserMapper, SysUserEntity>(), ISysUserService
